namespaceResourceQuotaResizing/nsQuotaResizing3_multipleContainerSpecs/README.md

```diff
@@ -0,0 +1,27 @@
+# Namespace Resource Quota Resizing With Multiple Container Specs
+
+Namespace has 2 container specs, each with one container. One has CPU highly utilized and the other has CPU under utilized:
+
+```
+Namespace:
+- CPU limit quota capacity: 1000m
+- CPU limit quota used: 1000m
+
+- ContainerSpec1:
+    - Container:
+        - CPU limit: 500m
+        - Desired CPU limit from Turbo analysis: 800m
+- ContainerSpec2:
+    - Container:
+        - CPU limit: 500m
+        - Desired CPU limit from Turbo analysis: 300m
+```
+
+- With namespace reisizing support, 
+    - in plan, as we simulate all container resizing actions, namespace quota resizing action will **NOT** be generated.
+    - in real time, we'll generate following actions:
+```
+- Resize namespace from 1000m to 1300m
+- Resize container1 from 500m to 800m (not executable)
+```
```

Comment:
Just a small comment about this particular scenario - it's hard to guarantee that the resize up and down will always be identical amounts to cancel each other out in a plan because if they are deployed on different nodes, the expense side of the ROI calculation may be different enough that one may go up by +300m and the other only goes down by -200m etc.

I think the scenario is fine to model a situation where we have a resize up and down in the same namespace where the quota is full, but I don't think it would be a good idea to have an automated test to expect exactly matching values on both sides.

Reply:
Good point! Updated the readme as:

> ### Note
> Note that this is to mainly test the scenario where there're both container resize up and down actions in the same namespace with resource quota full. The amount of resizing up and amount of resizing down may not be the same depending on the underlying nodes. The guaranteed result is:
> - in real time, namespace resizing up will be generated by the total amount of container resizing up;
> - in plan, namespace resizing up may or may not be generated based on the total amount of container resizing up and down.